src/res/translation/translation_pl_PL.ts

```diff
@@ -952,7 +952,11 @@
     <message>
         <location filename="../../clientdlgbase.ui" line="584"/>
         <source>Set up your audio, connect to a server and start jamming!</source>
+<<<<<<< HEAD
```

Comment:
This is a failed rebase which needs resolving.

Reply:
Sorry for trouble, next shoot should be correct.

Reply:
Thanks!